src/main/java/Duke.java

```diff
@@ -1,10 +1,272 @@
+     */
+    public static void printGreet() {
+        printDivider();
+        System.out.println(RABBIT_ASCII);
+        printDivider();
+        System.out.println(GREETING_MESSAGE);
+        printDivider();
+    }
+
+    /**
+     * Prints out the farewell message to standard output
+     */
+    public static void printFarewell() {
+        System.out.println(FAREWELL_MESSAGE);
+        printDivider();
+    }
+
+    /**
+     * Takes the provided input string and saves it into the array of Task "tasks"
+     * If the number of tasks exceed MAX_COUNT, the input string will not be saved.
+     *
+     * @param input    Input string to be saved
+     * @param taskType Type of task to save as
+     */
+    public static void saveInput(String input, Task.TaskType taskType) {
+        if (inputCount >= MAX_COUNT) {
+            System.out.println(ERROR_INPUT_LIMIT_REACHED +
+                    "The last entry, \"" + input + "\", will not be added to the list.");
+        } else {
+            String[] inputs;
+            boolean isTaskNameOnly;
+            switch (taskType) {
+            case TODO:
+                tasks[inputCount] = new Todo(input);
+                break;
+            case EVENT:
+                inputs = input.split(REGEX_PATTERN_AT, 2);
+                isTaskNameOnly = inputs.length < 2;
+                if (isTaskNameOnly) {
+                    System.out.println(ERROR_MISSING_EVENT_DELIMITER);
+                    return;
+                } else {
+                    tasks[inputCount] = new Event(inputs[0], inputs[1]);
+                }
+                break;
+            case DEADLINE:
+                inputs = input.split(REGEX_PATTERN_BY, 2);
+                isTaskNameOnly = inputs.length < 2;
+                if (isTaskNameOnly) {
+                    System.out.println(ERROR_MISSING_DEADLINE_DELIMITER);
+                    return;
+                } else {
+                    tasks[inputCount] = new Deadline(inputs[0], inputs[1]);
+                }
+                break;
+            default:
+                tasks[inputCount] = new Task(input);
+                break;
+            }
+            tasks[inputCount].printAddedMessage();
+            inputCount++;
+        }
+    }
+
+    /**
+     * Prints out the full list of inputs given by the user
+     */
+    public static void printTaskList() {
+        System.out.println("| Task List |");
+        if (inputCount == 0) {
+            System.out.println(ERROR_EMPTY_LIST);
+        } else {
+            for (int i = 0; i < inputCount; i++) {
+                System.out.println((i + 1) + ". " + tasks[i].toString());
+            }
+        }
+    }
+
+    /**
+     * Modifies the task identified by the input task number and print out a confirmation message.
+     *
+     * @param shouldMarkTask Determines whether to set task to done or not done
+     * @param taskNoString   The input task number that identifies the task in "tasks"
+     */
```

Comment:
I like that you have written documentation to help readers to understand the methods better